src/index.ts

```diff
@@ -120,6 +120,45 @@ window.XMLHttpRequest.prototype.open = function (
   return originalXhrOpen.apply(this, arguments as any);
 };
 
+const oldFetch = window.fetch;
+
+window.fetch = function fetch() {
```

Comment:
Do you guys add/handle DT headers elsewhere?

Reply:
i added it now.